cmd/describe/autoscaler/cmd.go

```diff
@@ -0,0 +1,66 @@
+package autoscaler
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/clusterautoscaler"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	use     = "autoscaler"
+	short   = "Show details of the autoscaler for a cluster"
+	long    = short
+	example = ` # Describe the autoscaler for cluster 'foo'
+rosa describe autoscaler --cluster foo`
+)
+
+func NewDescribeAutoscalerCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     use,
+		Short:   short,
+		Long:    long,
+		Example: example,
+		Args:    cobra.NoArgs,
+		Run:     rosa.DefaultRunner(rosa.DefaultRuntime(), DescribeAutoscalerRunner()),
+	}
+
+	output.AddFlag(cmd)
+	ocm.AddClusterFlag(cmd)
+	return cmd
+}
+
+func DescribeAutoscalerRunner() rosa.CommandRunner {
```

Comment:
Command logic is executed by a `CommandRunner`. The runner does not `os.Exit`, but instead returns an `error` if something went wrong. This makes it fully testable. The passed `Runtime` can be instantiated with mocks in the tests.